### PR TITLE
Split out fastdotcom into a sensor and component

### PR DESCRIPTION
**Pull request by rohankapoorcom**
**Description:**
Fast.com is now a component with a sensor platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20341

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.fastdotcom.markdown [hunk 1]
```diff
@@ -14,58 +14,15 @@ ha_release: 0.26
 ha_iot_class: "Cloud Polling"
 ---
 
-The `fastdotcom` sensor component uses the [Fast.com](https://fast.com/) web service to measure network bandwidth performance.
+The `fastdotcom` sensor uses the [Fast.com](https://fast.com/) web service to measure network bandwidth performance.
 
 <p class='note'>
-Currently fast.com only supports measuring download bandwidth. If you want to measure bandwidth metrics other then download such as ping and upload, utilize the [speedtest](/components/sensor.speedtest) component.
+You must have the [Fast.com component](/components/fastdotcom/) configured to use this sensor.
```
Comment (DubhAd) [inline source/_components/sensor.fastdotcom.markdown]: Not sure this needs to be a note. Maybe also worth rewording to say that the sensor is created automatically when the component is configured.

File: source/_components/fastdotcom.markdown [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+fastdotcom:
+```
+
+Every half hour of every day:
+
+```yaml
+fastdotcom:
+  update_interval:
+      minutes: 30
+
```
Comment (DubhAd) [inline source/_components/fastdotcom.markdown]: Stray blank line

**Review by DubhAd — CHANGES_REQUESTED** [review]

File: source/_components/fastdotcom.markdown [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+layout: page
+title: "Fast.com"
+description: "How to integrate Fast.com within Home Assistant."
+date: 2019-02-01 21:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: fastdotcom.png
+ha_category: System Monitor
```
Comment (klaasnicolaas) [inline source/_components/fastdotcom.markdown]:
```suggestion
ha_category: 
  - System Monitor
  - Sensor
```

File: source/_components/fastdotcom.markdown [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+featured: false
+ha_release: 0.88
+ha_iot_class: "Cloud Polling"
+---
```
Comment (klaasnicolaas) [inline source/_components/fastdotcom.markdown:19]:
```suggestion
redirect_from:
  - /components/sensor.fastdotcom/
---
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
We now work with a new way of component pages, so we no longer need to create a separate page for each sensor, switch, etc. but simply attach the information to the main page of the component and delete the sensor page file....

**Comment by rohankapoorcom** [comment]
>we no longer need to create a separate page for each sensor, switch, etc. but simply attach the information to the main page of the component and delete the sensor page file

@klaasnicolaas can you provide documentation/examples for this? Up until 1 week ago, PRs like this were being accepted: https://github.com/home-assistant/home-assistant.io/pull/8296 I'm a little confused when the change happened :)

File: source/_components/sensor.fastdotcom.markdown (hunk 1 above)
Comment (rohankapoorcom) [reply source/_components/sensor.fastdotcom.markdown]: It's a convention that was used for a lot of components that I split out (don't remember where I found it originally):

- https://rc--home-assistant-docs.netlify.com/components/device_tracker.locative/
- https://rc--home-assistant-docs.netlify.com/components/device_tracker.gpslogger/
- https://rc--home-assistant-docs.netlify.com/components/device_tracker.geofency/

The component indicates that the sensor will be created automatically: https://github.com/home-assistant/home-assistant.io/pull/8377/files/2b31fd8abc7d3b4d23f9f06de9310f5948c649fb#diff-176c5178582af3756262b510f5c8500fR19 Is it worth duplicating that statement here?

**Comment by klaasnicolaas** [comment]
@rohankapoorcom See this PR from Frenck: https://github.com/home-assistant/home-assistant.io/pull/8265

**Comment by rohankapoorcom** [comment]
@klaasnicolaas Thanks for the reference, consolidated down to a single component page.

File: source/_components/sensor.fastdotcom.markdown [hunk 5]
```diff
@@ -1,73 +0,0 @@
...
-description: "How to integrate Fast.com within Home Assistant."
-date: 2016-08-10 17:30
-sidebar: true
-comments: false
-sharing: true
-footer: true
-logo: fastdotcom.png
-ha_category: System Monitor
-featured: false
-ha_release: 0.26
```
Comment (rohankapoorcom) [inline source/_components/sensor.fastdotcom.markdown:13]: Should we keep the original `ha_release`?

Comment (klaasnicolaas) [reply source/_components/sensor.fastdotcom.markdown:13]: I think we should just leave it that way

**Review by frenck — APPROVED** [review]

**Pull request merged**